lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -228,6 +253,50 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Handle $this->_longData after regular query parameters have been bound
+     *
+     * @throws MysqliException
+     */
+    private function processLongData()
+    {
+        foreach ($this->longData as $paramNr => $value) {
+            if (! is_resource($value)) {
+                $this->sendLongData($paramNr, $value);
```

Comment:
Please bind strings as strings. Even if a LOB parameter is bound by reference, its value is already known by the moment `execute()` is called.